pkg/cloudprovider/provider/kubevirt/types/types.go

```diff
@@ -76,6 +76,9 @@ type Template struct {
 // PrimaryDisk.
 type PrimaryDisk struct {
 	Disk
+	// StorageTarget describes which VirtualMachine storage target will be used in the DataVolumeTemplate.
+	StorageTarget providerconfigtypes.ConfigVarString `json:"storageTarget,omitempty"`
```

Comment:
please add some tests in `provider_test.go`

Reply:
Done https://github.com/kubermatic/machine-controller/pull/1855/commits/40c283c65384a8482ff2a3c18832997fd1d96c32